01 - Community/Obsidian Roundup/🌲 Don't Minimize Difficulty.md

```diff
@@ -0,0 +1,15 @@
+![🌲 Don't Minimize Difficulty](https://www.obsidianroundup.org/content/images/2022/08/pexels-jadson-thomas-1500610.jpg)
```

Comment:
The presence of this file in this PR is a good example of why it's important to create Pull Requests from branches rather than from main.

This PR is asking to merge in from a `main` branch, and that branch contains this file, which was discovered in #492 to not be wanted. See that closed-un-merged PR for the reasons why.

But because this PR was also created from the same repo's `main`, it has this file in, and so all subsequent requests to merge from the same main will keep trying to push this file.